docs/guide/content_rendering/twig_function_reference/other_twig_filters.md

```diff
@@ -14,3 +14,32 @@
     <use xlink:href="{{ ibexa_icon_path('edit', 'my_icons') }}"></use>
 </svg>
 ```
+
+### Icons size variants
+
+The default icon size in the Back Office is `32px`. To change the default size, in the template add the modifier to the class name.
+
+``` twig
+<svg class="ibexa-icon ibexa-icon--medium">
+  <use xlink:href="{{ ibexa_icon_path('create') }}"></use>
+</svg>
+```
+
+The list of available icon sizes:
+
+|Size|Class postfix (modifiers)|
+|----|---------|
+|`8px`|`--tiny`|
+|`12px`|`--tiny-small`|
+|`16px`|`--small`|
+|`20px`|`--small-medium`|
+|`24px`|`--medium`|
+|`38px`|`--medium-large`|
+|`48px`|`--large`|
+|`64px`|`--extra-large`|
+
+!!! note
+
+	The values are defined in `rem` units and almost every `px` unit is converted to `rem`
+	using the following formula: <br>
+	`8px / (16px/rem) = 0.5rem`
```

Comment:
My formula in comment https://github.com/ezsystems/developer-documentation/pull/1577/files#r848225811 was simplified. 
Instead of `16px` there should be the base font size, which for our system is equal to `16px`, but can be changed to a different value.
I think that the icons' size section is not the right place to explain this.